<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="make_order">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Заказать</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="menu">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>261</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>111</width>
      <height>132</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox1">
       <property name="text">
        <string>Чизбургер</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox2">
       <property name="text">
        <string>Кола</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox3">
       <property name="text">
        <string>Нагетсы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox4">
       <property name="text">
        <string>Гамбургер</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>61</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit3"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit4"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
